Abbreviate license name

tobie opened the pull request:
This adds an `<abbr>` for the license name and displays the license shortname instead (CC BY-NC-SA 4.0).

tobie committed:
Abbreviate license name

aardrian reviewed (commented):
Because the `<abbr>` element is not exposed to assistive technology, and the `title` attribute does not support (non-IE/Edge) keyboard users, best practice is to define the term and then put the abbreviation in parentheses afterward. Like old-school print. Doing this on mobile while on a break from shoveling snow, so no idea if this transcribed properly. Will see if I can edit when in front of computer.

aardrian committed:
Abbreviation for CC BY-NC-SA 4.0

This follows the current best practices (insofar as a bunch of accessibility practitioners do this based on testing with users and the behavior of browsers) for providing abbreviations.

tobie committed:
Fix changes
